cog/admin.py

```
@@ -20,6 +20,54 @@ async def reload(self, ctx, package):
+        # 斜線指令的輸入框不能換行，用 \n 代替
+        text = content.replace("\\n", "\n")
+        # 「# 」開頭的行會顯示成 Discord 的大字標題
+        text = f"## 中電喵公告\n{text}"
+        if ping in ("@here", "@everyone"):
+            text = f"{ping}\n{text}"
+
+        try:
+            message = await channel.send(text)
+        except discord.Forbidden:
+            await ctx.respond(
+                f"我沒有在 {channel.mention} 發言的權限！", ephemeral=True
```

Ruff (0.15.20)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)